Makefile

```diff
@@ -3,3 +3,6 @@ include boilerplate/generated-includes.mk
 .PHONY: boilerplate-update
 boilerplate-update:
 	@boilerplate/update
+
+.PHONY: build-push
```

Comment:
Why is this necessary? It's already in [the convention's standard.mk](https://github.com/openshift/osd-metrics-exporter/blob/main/boilerplate/openshift/golang-osd-operator/standard.mk#L172-L174)

Reply:
Because the `build-push` in the convention only builds and pushes the operator image but not the catalog image.

Reply:
My bad, looks like it does.